examples/guestbook/frontend-service.json

```diff
@@ -17,6 +17,7 @@
       ],
       "selector":{
          "name":"frontend"
-      }
+      },
+      "createExternalLoadBalancer": true
```

Comment:
I removed the external load balancer change from this pr.
I'd still like it to be included in example script, but not before we can make sure that e2e won't be affected.